cannon/testdata/example/alloc/go.mod

```diff
@@ -1,8 +1,8 @@
 module alloc
 
-go 1.21
+go 1.22
```

Comment:
I don't think we can bump cannon due to changes in the Go abi, it will cause the i/o to break and need to be modified

Reply:
We can't bump op-program until cannon is updated to handle the new sys call go 1.22 issues either.

Reply:
The cannon tests will pick that up, but they're slow so don't run pre-merge.

Reply:
Yeah this was a mistake I've just missed removing it. Thanks for checking!